### PR TITLE
Update disabled property behavior for <TextLink> component

### PR DESCRIPTION
**Pull request by malcolmmadsheep**
### What changed

`<TextLink>` component's `disabled` prop behavior.

### Why it changed

`disabled` props doesn't work correct.

---

- [ ] Change is tested
- [ ] Change is documented
- [ ] Build is green
- [ ] Change of UX is approved by [wuwa](https://github.com/wuwa) or [milkyfruit](https://github.com/milkyfruit)

### CODE REVIEW

File: src/BaseComponents/TextLink/TextLink.spec.js [hunk 1]
```diff
@@ -111,26 +111,49 @@ describe('TextLink', () => {
...
+
+  describe('given `onClick` with `disabled` property', () => {
+    it('should call `preventDefault` and not call `onClick` without `link`', () => {
+      const onClick = spy();
+      const preventDefault = spy();
+      const driver = createDriver(<TextLink onClick={onClick} disabled/>);
+
+      driver.click({preventDefault});
+
+      expect(preventDefault.calledOnce).toBe(true);
```
Comment (lbelinsk) [inline src/BaseComponents/TextLink/TextLink.spec.js]: nice 👍

**Comment by lbelinsk** [comment]
@malcolmmadsheep thanks for the pr!
Check your style here in the storybook-pr: https://wix-wix-style-react-pr-1798.surge.sh/?selectedKind=5.%20Buttons&selectedStory=5.8%20Text%20Link&full=0&addons=0&stories=1&panelRight=0

You should change the cursor in disabled state to be default and not pinter

**Comment by lbelinsk** [comment]
@milkyfruit is this the correct color for the disabled?
I remember it was changed a while ago..
https://wix-wix-style-react-pr-1798.surge.sh/?selectedKind=5.%20Buttons&selectedStory=5.8%20Text%20Link&full=0&addons=0&stories=1&panelRight=0

**Comment by milkyfruit** [comment]
@lbelinsk yes, the disabled color is correct (D55)

**Comment by lbelinsk** [comment]
@malcolmmadsheep ping. Waiting for the small fix.

**Comment by malcolmmadsheep** [comment]
@lbelinsk sorry, I just pushed with `-f`, but forgot to pull your changes. Was there anything important?

**Comment by lbelinsk** [comment]
I don't remember :)
Probably small stuff.
Will review again

File: src/BaseComponents/TextLink/TextLink.spec.js [hunk 2]
```diff
@@ -102,7 +102,7 @@ describe('TextLink', () => {
     expect(driver.getTarget()).toBe('_blank');
   });
 
-  it('should call the onClick func when clicked', () => {
+  it('should call the `onClick` when clicked', () => {
```
Comment (lbelinsk) [inline src/BaseComponents/TextLink/TextLink.spec.js]: "the onclick" --> "onClick"

**Pull request merged**